controllers/leds.py

```diff
@@ -3,30 +3,36 @@
 from components.chassis import Chassis
 from controllers.shooter import ShooterController
 from components.leds import StatusLights, LedColours
+from components.vision import Vision
 
 
 class LedController:
     indexer: Indexer
     chassis: Chassis
     shooter: Shooter
     shooter_control: ShooterController
-    leds: StatusLights
+    status_lights: StatusLights
+    vision: Vision
 
     def execute(self):
-        if not self.indexer.has_cargo_in_chimney() and not self.leds.is_flashing:
-            self.leds.flash(LedColours.ORANGE)
+        if not self.vision.is_ready():
+            if not self.status_lights.is_flashing:
+                self.status_lights.flash(LedColours.PINK.value)
```

Comment:
The controller shouldn't care that a colour has a value.
```suggestion
                self.status_lights.flash(LedColours.PINK)
```